Update_Balance.py

```diff
@@ -0,0 +1,27 @@
+import CalcSeparators
+
+def Update_Balance():
+    userID = int(input("Please enter your user ID: "))
+    with open(r"accountfiles.txt","r") as file:
+        f = file.readlines()
+        if userID > len(f) or userID < 1:
+            print("Error: That User ID does not exist")
+            return
+        details = f[userID-1]
+        #This function calculates the positions of the separators in the line of text
+        Separators = CalcSeparators.CalcSeparators(userID,details)
+        if len(Separators) == 0:
+            print("Error: That User ID does not exist")
+            return
+        Balance = float(details[Separators[1]+1:Separators[2]])
+        print("Your balance is: "+str(Balance))
+        answer = float(input("Enter the new balance: "))
+        Balance = answer
+        details = details.replace(details[Separators[1]+1:Separators[2]],str(Balance))
+        print(details)
+        f[userID-1] = details
+        #Write changes to file
+        with open(r"accountfiles.txt", 'w') as fp:
+            # iterate each line
+            for number, line in enumerate(f):
+                fp.write(line)
```

Comment:
The code looks really good, just some conventions error and clean code thing, with the changes it will be even cleaner.
Btw, smart way to do the Separator finding.
Be sure that you have every functions running for Wednesday and last thing but not least: Good job Rachel!!!
(After demo this will be our v1.0.0, the next one will be harder)

Reply:
Thanks so much! I will make all the changes and complete the last functions